docs/guides/rule_based_profilers/how-to-create-a-new-expectation-suite-using-rule-based-profilers.md

```diff
@@ -0,0 +1,114 @@
+---
+title: How to create a new Expectation Suite using Rule Based Profilers
+---
+import Prerequisites from '../connecting_to_your_data/components/prerequisites.jsx';
+
+<Prerequisites>
+
+- Have configured a Datasource to connect to your data
+- Have a basic understanding of [Metrics in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/metrics.html)
+- Have a basic understanding of [Expectation Configurations in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/expectations/expectations.html#expectation-concepts-domain-and-success-keys)
+
+</Prerequisites>
+
+
+### Profilers
+
+A Profiler uses an Execution Engine to build a new Expectation Suite. It can use zero, one, or more Batches of data to decide which Expectations to include in the new Suite. It can use other information, like a schema or an idea. A profiler may be used to create basic high-level expectations based on a schema even without data, to create specific Expectations based on team conventions or statistical properties in a dataset, or even to generate Expectation Suites specifically designed to be rendered by a Descriptive Renderer for data discovery.
```

Comment:
@talagluck How can a profiler use 0 batches of data?..

Reply:
I'm pulling out this section entirely. @cdkini pointed out that making this doc too much about general profiling might add some confusion and necessitate a lot more documentation. Instead, I'm treating it as a doc purely about Rule-based Profiling, with a link to our general Profiling docs.

Reply:
Let's maybe discuss this live? Perhaps there's a happy medium we can achieve.